prompb/io/prometheus/write/v2/types.proto

```diff
@@ -31,6 +31,13 @@ import "gogoproto/gogo.proto";
 // are not part of the spec proto (they only modify the generated Go code, not
 // the serialized message). See: https://github.com/prometheus/prometheus/issues/11908
 message Request {
+  // Since Request supersedes 1.0 spec's prometheus.WriteRequest, we reserve the top-down message
+  // for the deterministic interop between those two, see types_test.go for details.
+  // Generally it's not needed, because Receivers must use the Content-Type header, but we want to
+  // be sympathetic to adopters with mistaken implementations and have deterministic error (empty
+  // message if you use the wrong proto schema).
+  reserved 1 to 3;
```

Comment:
makes sense, should we at the same time add an addtional field with some really high field value (is that possible? like `999`, to incidate the previous
```
// Cortex uses this field to determine the source of the write request.
  // We reserve it to avoid any compatibility issues.
  reserved  2;
  ```
   should use that field now in rw2.0?

Reply:
Sorry didn't get it, to indicate what?

Reply:
We discussed online - I don't have context on why Cortex needed or still need it. Does Mimir needs this too? 

However:
1. I think they can still use 2 since it's reserved as it was? (: 
1. We can always add reserved new field later

Reply:
discussed on slack